c2corg_ui/templates/index.html

```diff
@@ -38,4 +38,20 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
     </div>
   </section>
   <app-feed id="feed"></app-feed>
+
+  <div class="float-buttons" ng-if="userCtrl.auth.isAuthenticated()">
+    <div class="float-button-container">
+      <div class="float-button" ng-click="feedCtrl.toggleFilters()" ng-class="{'personal-feed-on': feedCtrl.isPersonal}">
+        <span class="glyphicon glyphicon-user"></span>
+      </div>
+      <p class="float-button-text" x-translate ng-bind="feedCtrl.isPersonal ? 'Personal feed on' : 'Personal feed off'">Personal feed</p>
```

Comment:
Is it really not possible to mark those strings as to be translated instead of adding them to ``i18n.html``?
For instance
```
<p class="float-button-text" x-translate ng-bind="feedCtrl.isPersonal ? ('Personal feed on' | translate) : ('Personal feed off' | translate)">Personal feed</p>
```

What is this ``Personal feed`` string? Shouldn't it be added to the translated strings as well (simply by using ``translate`` instead of ``x-translate``?